src/main/resources/org/clulab/asist/grammars/items.yml

```diff
@@ -41,3 +41,36 @@ rules:
     pattern: |
       trigger = [lemma=/map/]
       owner: Entity? = >"nmod:poss"
+
+# -------- marker blocks ----------
+  - name: marker_block
+    priority: ${ rulepriority }
+    label: MarkerBlock
+    type: token
+    keep: true
+    pattern: |
+      [lemma=/(?i)marker block|marker/]
```

Comment:
This won't work, what's inside the `[ ]` has to be a single token.
maybe something like this?
```
[lemma=marker] [lemma=block]?
```

Reply:
thanks!